{{cookiecutter.root_dir__update_if_you_intend_to_use_monorepo}}/{{cookiecutter.project_name}}/training/notebooks/Train.py

```diff
@@ -16,16 +16,15 @@
 #
 # Parameters:
 #
-# * env (optional): Name of the environment the notebook is run in (dev, staging, or prod). Defaults to "dev".
+# * env (optional): Name of the environment the notebook is run in (test, staging, or prod).
```

Comment:
Why not leave dev here? It will be used when the notebook is triggered manually, right?

Reply:
We are getting the `env` field without defining the field in the widget, which means only terraform jobs or github actions will be able to set the `env` parameter.

If users directly run the notebook, they will direction use the `dev` profile without being able to set the `env` field.